web3/account.py

```diff
@@ -0,0 +1,75 @@
+
+import functools
+import os
+
+from cytoolz import (
+    compose,
+)
+
+from eth_keys import KeyAPI
+
+from eth_utils import (
+    keccak,
+)
+
+import rlp
+
+from web3.module import Module
+
+from web3.utils.encoding import (
+    to_bytes,
+    to_decimal,
+)
+from web3.utils.signing import (
+    signature_wrapper,
+)
+
+
+def return_key_api(to_wrap):
+    @functools.wraps(to_wrap)
+    def wrapper(*args, **kwargs):
+        key = to_wrap(*args, **kwargs)
+        sign = key.sign
+        key.address = key.public_key.to_checksum_address()
+        key.privateKey = key
+        key.sign = compose(sign, signature_wrapper, to_bytes)
+        key.signTransaction = compose(sign, to_bytes)
+        return key
+    return wrapper
+
+
+class Account(Module):
+    _keys = KeyAPI()
```

Comment:
This has been updated so that you can just do `from eth_keys import keys` and access all the apis from the `keys` instance.